### PR TITLE
Feature/show name

### PR DESCRIPTION
**Pull request by yannhn**
added mock-name to display it on homepage. Added basic styling.

### CODE REVIEW

File: src/components/Layout.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import Footer from './Footer';
-import Header from './Header';
+// import Footer from './Footer';
+// import Header from './Header';
```
Comment (JessicaLoers) [inline src/components/Layout.js:2]: Please remove the import you're not using. 🙂

File: src/components/Layout.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 export default function Layout({children}) {
 	return (
 		<>
-			<Header />
+			{/* <Header /> */}
```
Comment (JessicaLoers) [inline src/components/Layout.js:7]: Please remove the comments for the unused components.

File: src/pages/index.js [hunk 3]
```diff
@@ -9,7 +10,7 @@ export default function HomePage() {
 				<title key="title">My Project</title>
 				<meta key="description" name="description" content="This is my project" />
 			</Helmet>
-			<h1>Home</h1>
+			<ShowName></ShowName>
```
Comment (JessicaLoers) [inline src/pages/index.js:13]: Yo only need to write <ShowName></ShowName> when the component has child components. It still works , but generally <ShowName/> is preferred. 🤓

**Review by JessicaLoers — COMMENTED** [review]
Lgtm. 👍 Just a few minor comments.

**Comment by yannhn** [comment]
Thanks for the feedback! I already implemented the feedback.

**Pull request merged**